rescript/orient.py

```diff
@@ -100,11 +102,12 @@ def _vsearch_revcomp_fastq(seqs_fp, out_fp):
     run_command(['gzip', str(out_fp)])
 
 
+# adapted from q2-demux
 def read_fastq(filepath):
-    # adapted from q2-demux
     fh = gzip.open(filepath, 'rt')
-    for header, seq, qh, qual in zip_longest(*[fh] * 4):
-        yield (header.strip(), seq.strip(), qh.strip(), qual.strip())
+    # vsearch strips off the lane info, so we will do the same for consistency
+    for head, seq, qh, q in zip_longest(*[fh] * 4):
+        yield (head.strip().split(' ')[0], seq.strip(), qh.strip(), q.strip())
 
 
 def orient_reads(
```

Comment:
Set `reference_sequences: DNAFASTAFormat = ` line. At the moment `--i-reference-sequences` appears as an optional setting in the plugin.